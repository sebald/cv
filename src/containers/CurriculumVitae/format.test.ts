import {
  formatActivityAtLocation,
  formatDateString,
  formatPeriod,
} from './format';

test('format date', () => {
  expect(formatDateString('2000-01')).toMatchSnapshot();
  expect(formatDateString('2017-12')).toMatchSnapshot();
  expect(formatDateString('2017-06-03')).toMatchSnapshot();
});

test('format period', () => {
  expect(formatPeriod('2000-01')).toMatchSnapshot();
  expect(formatPeriod('2000-01', '2000-05')).toMatchSnapshot();
});

test('format activity', () => {
  expect(formatActivityAtLocation('only a activity'));
  expect(formatActivityAtLocation('activitiy', 'location')).toMatchSnapshot();
});
