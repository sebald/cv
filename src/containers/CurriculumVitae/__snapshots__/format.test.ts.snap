// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format activity 1`] = `"activitiy @ location"`;

exports[`format date 1`] = `"Jan 2000"`;

exports[`format date 2`] = `"Dec 2017"`;

exports[`format date 3`] = `"Jun 2017"`;

exports[`format period 1`] = `"Jan 2000 - heute"`;

exports[`format period 2`] = `"Jan 2000 - May 2000"`;
